src/tmpfiles/tmpfiles.c

```diff
@@ -2633,6 +2750,50 @@ static int find_gid(const char *group, gid_t *ret_gid, Hashmap **cache) {
         return name_to_gid_offline(arg_root, group, ret_gid, cache);
 }
 
+static int parse_age_by_from_arg(const char *age_by_str, AgeBy *ret_age_by) {
+        const char *s = NULL;
+        AgeBy ab = 0;
+
+        static const struct {
+                char age_by_chr;
+                AgeBy age_by_flag;
+        } age_by_types[] = {
+                { 'a', AGE_BY_ATIME_F },
+                { 'b', AGE_BY_BTIME_F },
+                { 'c', AGE_BY_CTIME_F },
+                { 'm', AGE_BY_MTIME_F },
+                { 'A', AGE_BY_ATIME_D },
+                { 'B', AGE_BY_BTIME_D },
+                { 'M', AGE_BY_MTIME_D },
```

Comment:
where did C go?

Reply:
`C` is ignored for directories (well, I see (below) that we want to allow it as an argument, but ignore it when `dir_cleanup()`).